### PR TITLE
liteide: fix download url

### PR DESCRIPTION
**Pull request by issaclin32**
#2308 (Outstanding Excavator issues)

### CODE REVIEW

**Comment by Ash258** [comment]
Why there are some junk commits from third of August in PR?

Create new clean branch from upstream master

File: bucket/liteide.json [hunk 1]
```diff
@@ -18,6 +18,6 @@
         "regex": "tag/x([\\d\\.]+)"
     },
     "autoupdate": {
-        "url": "https://downloads.sourceforge.net/project/liteide/x$version/liteidex$version.windows-qt5.9.5.zip"
+        "url": "https://github.com/visualfc/liteide/releases/download/x36.2/liteidex$version.windows-qt5.9.5.zip"
```
Comment (linsui) [inline bucket/liteide.json]:
```suggestion
        "url": "https://github.com/visualfc/liteide/releases/download/x$version/liteidex$version.windows-qt5.9.5.zip"
```

Comment (issaclin32) [reply bucket/liteide.json]: Forgot to change it. :sweat_smile:. Thanks.

**Comment by issaclin32** [comment]
> Why there are some junk commits from third of August in PR?
> 
> Create new / Always start PR with clean branch from upstream master

Okay. I will re-start the PR to clean those junk commits.

**Pull request closed without merging**